src/ui/events.js

```diff
@@ -778,6 +778,20 @@ export type MapEvent =
      */
     | 'sourcedataloading'
 
+    /**
+     * Fired when an icon or pattern needed by the style is missing. The missing image can
+     * be added with {@link Map#addImage} within this callback to prevent the image from
```

Comment:
Maybe better to be explicit here -> `within the event listener callback`

Reply:
yep, good idea!